typespec-tests/tsp/response.tsp

```diff
@@ -203,4 +203,14 @@ interface ResponseOp {
 
   // @route("/paged-datetime")
   // listDateTimes(): DateTimesList;
+
+  @route("/json-utf8-response")
+  getJsonUtf8Response(): Resource & {
+    @header contentType: "application/json;charset=utf-8";
+  };
+
+  @route("/plus-json-response")
+  getPlusJsonResponse(): Resource & {
+    @header contentType: "application/vnd.microsoft.appconfig.kv+json";
+  };
```

Comment:
https://datatracker.ietf.org/doc/html/rfc6839#section-3.1

commented in code as well